keps/sig-storage/1488-csi-migration-gce-pd/kep.yaml

```diff
@@ -19,7 +19,7 @@ status: implementable
 see-also:
   - "https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/csi-migration.md"
 prr-approvers:
-  - "@wojtek-t"
+  - "@msau42"
```

Comment:
Since PRR approver is in a separate file now, we no longer need this here.  You can remove lines 21-22.
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L23

Reply:
ah, thanks, done